base-template.sh

```diff
@@ -518,38 +583,88 @@ update_shared_hooks_if_appropriate() {
 
     RUN_UPDATE="false"
     [ "$HOOK_NAME" = "post-merge" ] && RUN_UPDATE="true"
-    [ "$HOOK_NAME" = ".githooks.shared.trigger" ] && RUN_UPDATE="true"
     [ "$HOOK_NAME" = "post-checkout" ] && [ "$1" = "$GIT_NULL_REF" ] && RUN_UPDATE="true"
 
     if [ "$RUN_UPDATE" = "true" ]; then
+
         # split on comma and newline
-        IFS="$IFS_COMMA_NEWLINE"
+        IFS_TMP="$IFS_COMMA_NEWLINE"
+        [ "$SHARED_HOOKS_TYPE" = "--shared" ] && IFS_TMP="$IFS_NEWLINE"
```

Comment:
Does this mean the local/global git config can be comma or newline, but the entries in the `.shared` file will not be split on commas? (only line by line)
Fine if that is the case, just not obvious from reading this.

Reply:
Thx, this is a wrong left over (:see_no_evil: we have so many duplicates of functionality hm...)
Of course with **this** PR -> only IFS_NEWLINE required -> since everything is in multiple git config values and ->
.shared only supports newline (no commas).